mosquitto/rootfs/etc/cont-init.d/mosquitto.sh

```diff
@@ -50,8 +48,7 @@ for login in $(bashio::config 'logins|keys'); do
 
   bashio::log.info "Setting up user ${username}"
   password=$(np -p "${password}")
-  echo "${username}:${password}" >> "${PW}"
-  echo "user ${username}" >> "${ACL}"
+  sqlite3 "${DB}" "insert into users values('${username}','${password}');"
```

Comment:
There seems to be a level of escaping special chars missing ... 
e.g.
```text
does\test
```
in username 
results in 
```text
does    est
```

Reply:
Looks like this was also an issue before. Single quotes make it work
```yaml
  - username: 'does\test'
    password: '*#`1234%!&\$)(=)()=!§'
```